pkg/translator/loki/logs_to_loki.go

```diff
@@ -67,39 +68,21 @@ func LogsToLokiRequests(ld plog.Logs) map[string]PushRequest {
 			logs := ills.At(j).LogRecords()
 			scope := ills.At(j).Scope()
 			for k := 0; k < logs.Len(); k++ {
+				log := logs.At(k)
+				resource := rls.At(i).Resource()
 
-				// similarly, we may remove attributes, so change only our version
-				log := plog.NewLogRecord()
-				logs.At(k).CopyTo(log)
-
-				// we may remove attributes, so we make a copy and change our version
-				resource := pcommon.NewResource()
-				rls.At(i).Resource().CopyTo(resource)
-
-				// adds level attribute from log.severityNumber
-				addLogLevelAttributeAndHint(log)
+				entry, err := LogToLokiEntry(log, resource, scope)
```

Comment:
We should check for the error here. Now, this check is done [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18532/files#diff-d47b90a2367b146d22e0a67c51700f7c3908dd0b2b47423a3885f38f7f7f8784R86).

Reply:
in case of error, I still need to get the tenant from entry to create a group for that tenant and store error report below 
That's why LogToLokiEntry returns 
```
&PushEntry{
   Entry: nil,
   Labels: ..,
   Tenant: tenant,
}, err
```
in case of error